docs/manage-metalake-using-gravitino.md

```diff
@@ -136,9 +133,6 @@ GravitinoMetalake renamed = gravitinoAdminClient.alterMetalake(
 ```python
 changes = (
     MetalakeChange.rename("metalake_new_name"),
-    MetalakeChange.update_comment("metalake_new_comment"),
-    MetalakeChange.remove_property("metalake_properties_key1"),
-    MetalakeChange.set_property("metalake_properties_key2", "metalake_properties_new_value"),
 )
 
 metalake = gravitino_admin_client.alter_metalake("metalake_name", *changes)
```

Comment:
To be consistent with the java example, here the metalake name should be "metalake", what do you think?

Reply:
Yes, it would make more sense. I hesitated for a second and didn't change the values of the Python examples because they all adopted a different naming convention -- I'm afraid one minor update would bring more inconsistencies at this moment.

Reply:
I've synced the sample values in Python examples. @jerryshao thanks for your review.